pkg/services/auth/idimpl/service.go

```diff
@@ -97,6 +97,8 @@ func (s *Service) SignIdentity(ctx context.Context, id identity.Requester) (stri
 			},
 			Rest: authnlib.IDTokenClaims{
 				Namespace: s.nsMapper(id.GetOrgID()),
+				UID:       id.GetUID(),
```

Comment:
Not all entities have uid:s. Those are api-keys, render-key, anonymous and I guess provisioning. Any idea how we should handle those? Just set the normal typed id for them in uid field?

api-keys could have it but we are working on removing them completely so I don't think it is worth the effort, but the rest can never have a true uid.